zilliqa/src/consensus.rs

```diff
@@ -2592,6 +2589,10 @@ impl Consensus {
         &self.state
     }
 
+    pub fn state_mut(&mut self) -> &mut State {
```

Comment:
Why do we expose this? Its deliberate that `state` isn't `pub` to ensure `Consensus` is the only one who can mutate it.